doc/doxygen/images/alfeld-split.dia

```diff
@@ -0,0 +1,335 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you mean to commit this file? I don't think you reference it, and it's not clear to me what it represents.

Reply:
yes, this is the source file for the .svg for http://dia-installer.de/